### PR TITLE
Update footer and header

### PR DESCRIPTION
**Pull request by matiasilva**

### CODE REVIEW

File: control/templates/base.html [hunk 1]
```diff
@@ -1,115 +1,124 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
+<html lang="en" class="h-100">
```
Comment (edwinbalani) [inline control/templates/base.html:2]: `h-100` is a fix for the global stylesheet change.

File: control/templates/base.html [hunk 2]
```diff
@@ -1,115 +1,124 @@
...
+                                addresses</a></li>
+                        <li><a href="https://status.srcf.net"><i class="fa fa-li fa-toggle-on"></i> Server status</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <hr>
+            <p class="text-center text-muted">The Student-Run Computing Facility is a society at the University of
+                Cambridge run entirely by student volunteers &mdash; <a href="https://github.com/srcf"><i
+                        class="ml-2 fa fa-github"></i></a> </p>
```
Comment (edwinbalani) [inline control/templates/base.html:115]: This paragraph has been added to the footer.

File: control/templates/base.html [hunk 3]
```diff
@@ -1,115 +1,124 @@
...
+                <li class="nav-item"><a class="nav-link mr-1" href="{{ url_for('home.logout') }}"><i
+                            class="fa fa-sign-out"></i> <span class="d-none d-md-inline"> <strong>Log
+                                out</strong></span></a></li>
+            </ul>
         </div>
-      </div>
     </nav>
-    <main role="main" class="container">
-      {%- if page_title is defined %}
+    <main role="main" class="container flex-shrink-0">
```
Comment (edwinbalani) [inline control/templates/base.html:47]: `flex-shrink-0` is a fix for the global stylesheet change.

**Review by edwinbalani — COMMENTED** [review]
This review is not a review: just capturing details of the changes made.

Most of the diff is due to autoformatting -- text wrapping and indentation changes -- so the real updates are not so visible.

File: control/templates/base.html (hunk 1 above)
Comment (matiasilva) [reply control/templates/base.html:2]: Note to future readers too:

 * `h-100` combined with `mt-auto` allows the footer to create as much margin as it needs above it to make the nav sticky and nice without fixed values or heights

**Pull request merged**